sapi/phpdbg/phpdbg.h

```diff
@@ -44,6 +44,7 @@
 #include "zend_ini_scanner.h"
 #include "zend_stream.h"
 #include "zend_signal.h"
+#include "Optimizer/zend_dump.h"
```

Comment:
I don't think this should be included in phpdbg.h, rather the places that actually use it.

Reply:
Little bit of refactoring follows.